frontend/views/components/Transitions/MaskToModal/mixinTransitions.js

```diff
@@ -85,13 +85,13 @@ const animationMixins = {
       // And Fade in the targetInner only after the Masker has completed its transition animation
       Velocity(targetInner, { opacity: 1 }, { duration: this.config.fade, delay: this.config.delay, complete })
     },
-    transitionAfterEnter (el) {
+    transitionAfterEnter (el: HTMLElement) {
       console.log('transitionAfterEnter')
       // Trigger/Target has opacity: 0 by default, so let's force to stay 1 after the animation finishes.
       // Velocity(el, { opacity: 1 }, { duration: 0 })
-      el.style.opacity = 1
+      el.style.opacity = '1'
```

Comment:
Is this right? Did Flow say to change this?

Reply:
![flowError1](https://user-images.githubusercontent.com/17641213/55843923-b9eb0800-5b75-11e9-9f16-41cb7eea3114.png)

Yes, this is what flow spits out about that code and it seems to be referring to [this type defintion](https://github.com/facebook/flow/blob/master/lib/cssom.js#L278).

A possible solution that is suggested in [this issue](https://github.com/facebook/flow/issues/3733#issuecomment-295235903) actually fixes the error but I'm afraid it could make the code verbose just to keep flow happy.
(changing it to
const elStyle: Object = el.style
      elStyle.opacity = 1
)

Reply:
Hmm. OK, makes sense. I guess as long as what you have here works the same it's fine! :+1:

Reply:
Ok!